### PR TITLE
chore: Update dependencies

### PR DESCRIPTION
**Pull request by MichaelDeBoey**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,17 +27,14 @@
...
     "validate": "kcd-scripts validate"
   },
   "dependencies": {
-    "@babel/runtime": "^7.9.2"
+    "@babel/runtime": "^7.12.1"
   },
   "devDependencies": {
-    "all-contributors-cli": "^6.14.0",
-    "kcd-scripts": "^5.6.1"
+    "kcd-scripts": "^6.6.0"
```
Comment (robinmetral) [inline package.json:37]: What are the breaking changes here? Is it safe to merge or does it depend on the other PR?

**Review by robinmetral — COMMENTED** [review]
Thanks as always for your contributions @MichaelDeBoey 🙂

File: package.json (hunk 1 above)
Comment (MichaelDeBoey) [reply package.json:37]: It's safe to merge 🙂

**Pull request merged**